docs/_index_included.yaml

```diff
@@ -123,3 +123,15 @@ included:
     - heading: Experimental Wavefunctions
       description: Analyze published experimental wavefunctions.
       path: /cirq/experiments/qcqmc/experimental_wavefunctions
+
+- heading: Lattice Gauge Theory
+  description: Simulations for Charges and Strings in (2+1)D Lattice Gauge Theories
+  options:
+    - cards
+  items:
+    - heading: Experiment Example
+      path: /cirq/experiments/lattice_gauge/lattice_gauge
+    - heading: Code Layout
+
+
+docs/lattice_gauge/lattice_gauge.ipynb
```

Comment:
This looks like a leftover and also breaks yaml syntax. Please delete.

Reply:
Done.